### PR TITLE
add support for modifying request variables from a plugin

### PR DESCRIPTION
**Pull request by garypen**
Native or rhai.

fixes: #1141

### CODE REVIEW

**Review by o0Ignition0o — APPROVED** [review]
Super cool!

File: NEXT_CHANGELOG.md [hunk 1]
```diff
@@ -60,30 +60,11 @@ By [@bryncooke](https://github.com/bryncooke) in https://github.com/apollographq
 
 ## 🚀 Features ( :rocket: )
 
-### Add support of multiple uplink URLs [PR #1210](https://github.com/apollographql/router/pull/1210)
```
Comment (Geal) [inline NEXT_CHANGELOG.md:63]: these should not be removed from the next_changelog, right? Some entries are from main, some from api-1.0

File: NEXT_CHANGELOG.md [hunk 2]
```diff
@@ -60,30 +60,11 @@ By [@bryncooke](https://github.com/bryncooke) in https://github.com/apollographq
...
+### Add support for modifying variables from a plugin. [PR #1257](https://github.com/apollographql/router/pull/1257)
```
Comment (Geal) [inline NEXT_CHANGELOG.md:63]: these should not be removed from the next_changelog, right? Some entries are from main, some from api-1.0

**Review by Geal — CHANGES_REQUESTED** [review]

**Review by bnjjj — APPROVED** [review]

File: NEXT_CHANGELOG.md (hunk 2 above)
Comment (Geal) [reply NEXT_CHANGELOG.md:63]: right, that makes sense

**Pull request merged**